ctk/common/src/main/kotlin/org/codice/compliance/verification/core/NameIDPolicyVerifier.kt

```diff
@@ -46,22 +46,23 @@ class NameIDPolicyVerifier(private val samlResponseDom: Node, private val policy
         nameId.attributeText(SP_NAME_QUALIFIER)?.let { spnq ->
             val spNameQualifier = policy.spNameQualifier
             if (spnq != spNameQualifier) {
-                throw SAMLComplianceException.create(SAMLCore_3_4_1_1_b,
-                        message = "A NameID element was found with a SPNameQualifier " +
-                                "attribute value of $spnq instead of " +
-                                "$spNameQualifier.",
-                        node = nameId)
+                Report.addExceptionMessage(CORE_3_4,
+                        SAMLComplianceException.create(SAMLCore_3_4_1_1_b,
+                                message = """A NameID element was found with a SPNameQualifier
+                                        attribute value of $spnq instead of $spNameQualifier.""",
```

Comment:
❓ Maybe I'm remembering incorrectly but isn't changing this to a raw string going to include the two lines, with a newline in between, in the output message?

Reply:
I'm not sure but I put it back to how it was